exports/scripts/find.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+
+"""See help fields inside _main or run find.py --help for information."""
+
+from __future__ import annotations
+
+import argparse
+import sys
+from fnmatch import fnmatchcase
+from pathlib import Path
+
+
+def _glob_paths(root_path: Path, patterns: list[str]) -> set[Path]:
+    paths: set[Path] = set()
+    for p in patterns:
+        for path in root_path.glob(p):
+            paths.add(path)
+    return paths
+
+
+def _find(
+    roots: list[str],
+    include_patterns: list[str],
+    exclude_patterns: list[str],
+    file_name_patterns: list[str],
+) -> int:
+    for root in roots:
+        root_path = Path(root)
+        if not root_path.exists():
+            sys.stderr.write(f"[error] Path does not exist: '{root_path}'\n")
+            return 1
```

Comment:
If we're doing binary returns, I would prefer we just return a boolean. (Saves time when reading the code since you don't need to check if there's a surprise third option.)